src/main/scala/cromwell/util/GcsBucketInfo.scala

```diff
@@ -0,0 +1,10 @@
+package cromwell.util
+
+import com.google.api.client.util.DateTime
+import com.google.api.services.storage.model.Bucket.Owner
+
+class GcsBucketInfo(bucketName: String, location: String, timeCreated: DateTime, owner: Owner)
+{
+  def getBucket = bucketName
```

Comment:
You should be able to make this a case class. Also you shouldn't need the getters here, they're generally frowned upon in scala